app/views/pictures/index.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.array!(@pictures) do |picture|
+  json.extract! picture, :id, :description
+  json.url picture_url(picture, format: :json)
+end
```

Comment:
Yes, I should have not had jbuilder in there.  I had hopes of using it (or at least spiking it), but I have not yet been able to do so (and don't think I will have time).  Better to just leave it out until it's is needed.  Thanks for reminder.  Also, I see it created a lot of code in my controllers that is now unused, which I also know is bad practice.  Another lesson learned (the hard way)!